test/parallel/test-bootstrap-modules.js

```diff
@@ -44,6 +44,7 @@ expected.beforePreExec = new Set([
   'NativeModule internal/assert',
   'NativeModule internal/util/inspect',
   'NativeModule internal/util/debuglog',
+  'NativeModule internal/util/trace_timer',
```

Comment:
This should only be loaded when the timer is enabled.

Reply:
`trace_timer` is also imported by `Console`, so that's why is loaded every time.

Do you want to lazy load even for `Console`?

Reply:
Done!